enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2870,3 +2867,28 @@ func getRandomNUMANode() (int, error) {
 
 	return nodes[rand.IntN(len(nodes))], nil
 }
+
+// Returns the paths relative to the workspace root that should be copied back
+// to the action workspace directory after execution has completed.
+//
+// For performance reasons, we only extract the action's declared outputs,
+// unless the action is running with preserve-workspace=true.
+func workspacePathsToExtract(task *repb.ExecutionTask) []string {
+	if platform.IsTrue(platform.FindEffectiveValue(task, platform.PreserveWorkspacePropertyName)) {
+		return []string{"/"}
+	}
+
+	// Special files
+	// TODO: declare this list as a constant somewhere?
+	paths := []string{
+		".BUILDBUDDY_DO_NOT_RECYCLE",
+		".BUILDBUDDY_INVALIDATE_SNAPSHOT",
```

Comment:
@maggie-lou FYI I preemptively added the new file here from your in-flight PR. After our PRs are both merged I'm thinking of maybe adding a list to `container.go` with these special file names